library/src/layout/table.rs

```diff
@@ -280,6 +295,7 @@ impl<T: Into<Value>> From<Celled<T>> for Value {
         match celled {
             Celled::Value(value) => value.into(),
             Celled::Func(func) => func.into(),
+            Celled::Array(arr) => arr.into(),
         }
```

Comment:
Lastly, I'm not sure if I can just call `arr.into()` in the `from()` function for `Value`. What does it exactly do?

Reply:
likely casts to `Value::Array(arr)`

Reply:
It was surprising to me that this works, since `arr` is of type `Vec<T>` and not a type from Typst. Maybe I'm just confused by the rust language.

Reply:
Oh, I didn't see it was a Vec. In that case, note that Typst has manually created a handy conversion implementation for Vec:

https://github.com/typst/typst/blob/b6a241cad232d920154986781c6f09d83dc1b6f6/src/eval/cast.rs#L256-L260

Reply:
Oooh now I see it. Thanks!